### DISCUSSION TITLE
FileSieve v4.61

### DISCUSSION THREAD

**Discussion by BootBlock** [BootBlock/FileSieve#102] [General]
FileSieve 4.61 has been released; change log is listed below. Ignore the `Source code` downloads as it's an auto-generated thing.

* Other: FileSieve is now a 64-BIT process; 32-BIT systems are no longer supported.
* Added: HEIC support to the EXIF method. [[more](https://github.com/BootBlock/FileSieve/issues/11)]
* Added: Additional JPG file extensions to the EXIF method.
* Added: Remove Unused Codes to the EXIF method.

<hr /><em>This discussion was created from the release <a href='https://github.com/BootBlock/FileSieve/releases/tag/'>FileSieve v4.61</a>.</em>

**Comment by ainuke** [comment]
I just have a question; sorry if this isn't the proper forum.
I installed 4.61 to see if FS would do what I wanted, which is limited to moving folders (with contents). I frequently buy assets that have to be copied into an existing file structure. They're delivered in this file structure, but usually buried at least a couple of folders deep. Usually as bob.zip>bob/content/folder1/folder2/files.files. Everything above folder1 is variable and discarded; folder1 and below is the file structure that gets copied to the destination. 

I'm hoping FileSieve can drill down and copy/move folder1 and below to the destination. I haven't figured out which of the settings to choose to move folders, or even if this is possible. In the past I've used RoboBasket, but it will only move files, not folders (and the dev has no plans to do this).

Any advice you have would be welcome, and I'd be thrilled to buy a home license if FS could be bent to my will...  :)
Thanks in advance
Erik
